forms-flow-web/src/components/Modeller/Editors/BpmnEditor/index.js

```diff
@@ -154,7 +148,8 @@ export default React.memo(
       // Deployment Source
       form.append("deployment-source", "Camunda Modeler");
       // Tenant ID
-      if (tenantKey && !applyAllTenants) {
+      //if REACT_APP_PUBLIC_WORKFLOW_ENABLED is false in .env add tenant key to keep it private
+      if (MULTITENANCY_ENABLED && tenantKey && publicWorkflowEnabled === "false" && isNewDiagram) {
```

Comment:
We wouldn't need this I think, just remove the checkbox if the feature is disabled, then everything should work as expected with current code.

Reply:
Yes but if we remove this code, then the tenant user can create only the private workflows. once the env is enabled true, then the user can create public workflows too.